src/NumberToWords.php

```diff
@@ -87,7 +90,8 @@ class NumberToWords
         'sk' => SlovakCurrencyTransformer::class,
         'tk' => TurkmenCurrencyTransformer::class,
         'tr' => TurkishCurrencyTransformer::class,
-        'ua' => UkrainianCurrencyTransformer::class
+        'ua' => UkrainianCurrencyTransformer::class,
+        'ka' => GeorgianCurrencyTransformer::class
```

Comment:
Keys in the array are sorted alphabetically. Could you please move the `'ka' => GeorgianCurrencyTransformer::class` above?